### DISCUSSION TITLE
Font size in clock-mode

### DISCUSSION THREAD

**Discussion by a-kam** [pixelit-project/PixelIt#110] [Q&A]
Hello! And thanks for this awesome project.
I'm wondering if it is possible to enlarge the numbers in clock mode, to make it look more fullscreen? Maybe getting rid of the short lines at the bottom of the matrix? Btw.: What are these lines for?
Thanks

**Comment by o0shojo0o** [comment]
Hi, the clock does not support any other font size at the moment, this would have to be programmed first.
The lines under the clock show the days of the week.
